### DISCUSSION TITLE
Doubling the size of the free text input fields

### DISCUSSION THREAD

**Discussion by dhoitink** [formbricks/formbricks#4467] [General Requests]
Hello lovely people of the FB commuity, 
Is there a way to add more vertical space for text input fields? I know i can stretch them by using the mouse, however on mobile this is often not possible. Also it gives the user the impression there is not much space to fill in...  
See attachetment f.e.  Thx!!

![inputfiieldsizesvertical](https://github.com/user-attachments/assets/a22e4b3c-5104-4a5f-b378-74308a6c3d54)

**Comment by gupta-piyush19** [comment]
Hey @dhoitink, It's not yet possible to set the default height or width of the text area. However, you can enable the max length for this open-text question and set a higher value for it to make it more intuitive.

**Reply by dhoitink** [reply]
Hi @gupta-piyush19 , thx for replying. I have enabled 'long answer' on 'free text'. However i don't see 'open text' as a field option. Or is this exclusive to the paid version? 
![Screenshot 2024-12-17 at 16 25 43](https://github.com/user-attachments/assets/69f2f02b-00cd-4057-907b-9f36900d5e06)
